### PR TITLE
Add VMs of a MIG to an AD group named after the MIG

### PR DESCRIPTION
**Pull request by idofl**

### CODE REVIEW

**Comment by idofl** [comment]
These are the commands we'll need to add to the doc (step 5):
https://cloud.google.com/solutions/configuring-active-directory-for-vms-to-automatically-join-the-domain#create_an_active_directory_user_account

& dsacls.exe $BaseOrgUnitPath /G $CompRegistrar":CCDC;Group" /I:T | Out-Null
& dsacls.exe $BaseOrgUnitPath /G $CompRegistrar":LC;;Group" /I:S  | Out-Null
& dsacls.exe $BaseOrgUnitPath /G $CompRegistrar":RC;;Group" /I:S  | Out-Null
& dsacls.exe $BaseOrgUnitPath /G $CompRegistrar":WD;;Group" /I:S  | Out-Null
& dsacls.exe $BaseOrgUnitPath /G $CompRegistrar":WP;;Group" /I:S  | Out-Null
& dsacls.exe $BaseOrgUnitPath /G $CompRegistrar":RP;;Group" /I:S  | Out-Null

File: ad-joining/register-computer/ad/domain.py [hunk 1]
```diff
@@ -212,6 +213,53 @@ def get_upn_by_samaccountname(self, samaccountname):
...
+        except ldap3.core.exceptions.LDAPNoSuchObjectResult as e:
+            raise NoSuchObjectException(e)
+
+    def add_group(self, ou, group_name):
+        try:           
+            dn = "CN=%s,%s" % (group_name, ou)
+            self.__connection.add(
+                dn,
+                [
+                "group",
```
Comment (jpassing) [inline ad-joining/register-computer/ad/domain.py:240]: You're not specifying the `groupType` - does this still create a domain-local security group?

Comment (idofl) [reply ad-joining/register-computer/ad/domain.py:242]: It was created as a global security group. I added groupType and set it domain-local security group.

**Comment by jpassing** [comment]
lgtm

**Pull request merged**